webAO/styles/client.css

```diff
@@ -311,17 +325,20 @@
 
 .client_bench {
 	position: absolute;
-	height: auto;
-	width: 100%;
+	height: 100%;
```

Comment:
some benches are only a picture of the bench and this way they would stretch the entire height of the viewport and block it

Reply:
![bancodefensa](https://github.com/AttorneyOnline/webAO/assets/10584181/a4ad89b0-a191-4dbe-990b-8bb4452c9dd9)